### PR TITLE
add icsexport action

### PR DESCRIPTION
**Pull request by tschanzt**

### CODE REVIEW

File: ftw/events/profiles/default/actions.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<object name="portal_actions" meta_type="Plone Actions Tool"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+    <object name="document_actions">
+        <object name="ics_export" meta_type="CMF Action"
+                i18n:domain="ftw.events">
+                <property name="title" i18n:translate="">ICS Export</property>
+                <property name="description" i18n:translate="">Allows to exort the event to a calendar app</property>
```
Comment (jone) [inline ftw/events/profiles/default/actions.xml]: What do you mean with `exort`? 😂

File: ftw/events/upgrades/20160915095729_add_ics_action/actions.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<object name="portal_actions" meta_type="Plone Actions Tool"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n">
+    <object name="document_actions">
+        <object name="ics_export" meta_type="CMF Action"
+                i18n:domain="ftw.events">
+                <property name="title" i18n:translate="">ICS Export</property>
+                <property name="description" i18n:translate="">Allows to exort the event to a calendar app</property>
```
Comment (jone) [inline ftw/events/upgrades/20160915095729_add_ics_action/actions.xml]: `exort` here too

File: ftw/events/upgrades/20160915095729_add_ics_action/actions.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+                <property name="url_expr">string:${here/absolute_url}/ics_view</property>
```
Comment (jone) [inline ftw/events/upgrades/20160915095729_add_ics_action/actions.xml]: Where is the `ics_view` implemented?
We should probably have a test where we click on this action using a testbrowser and see if we actually get an ics.

Comment (Tschanzt) [inline ftw/events/upgrades/20160915095729_add_ics_action/actions.xml]: @jone ics_view is provided by plone.app.event.

**Comment by tschanzt** [comment]
@jone I added the test.

**Comment by shylux** [comment]
@tschanzt cmd+f "exort"

**Pull request merged**